pkg/network/ndmz/dualstack.go

```diff
@@ -201,6 +259,17 @@ func (d *DualStack) IP6PublicIface() string {
 	return d.ipv6Master
 }
 
+// SupportsPubIPv4 implements DMZ interface
+func (d *DualStack) SupportsPubIPv4() bool {
+	return d.hasPubBridge
+}
+
+// EnsureRoutable implements DMZ interface
+func (d *DualStack) EnsureRoutable() error {
```

Comment:
This statement is not really true. You can be dualstack and also only have one physical interface where all the traffic goes and that is linked to the zos bridge.
The type of ndmz is not related to the fact that there are 2 interfaces for traffic (management and public).
You could perfectly be a hidden node that has 2 physical interfaces